docs/tutorials/ui.md

```diff
@@ -118,6 +118,9 @@ ui.configure(config);
  - Init segment contains `prji` and `hfov` boxes
 
 
+If you want the VR to be rendered outside of the main container, add the
+data-shaka-player-vr-canvas` tag to a canvas element on the page.
```

Comment:
You are missing a quote on the start of the tag name.

Reply:
Fixed!